packages/sirius-web/frontend/sirius-web-application/src/views/library-browser/useLibraries.types.ts

```diff
@@ -46,6 +48,7 @@ export interface GQLLibrary {
   version: string;
   description: string;
   createdOn: string;
+  currentEditingContext?: GQLEditingContext;
```

Comment:
If you need to make it nullable, make that explicit. It should not be nullable in this case.

Reply:
Done